paramak/parametric_neutronics/make_faceteted_neutronics_model.py

```diff
@@ -109,9 +109,9 @@ def find_all_surfaces_of_reflecting_wedge(new_vols):
         #area = surface.area()
         vertex_in_surface = cubit.parse_cubit_list("vertex", " in surface " + str(surface_id))
         if surface.is_planar() == True and len(vertex_in_surface) == 4:
-        surface_info_dict[surface_id] = {'reflector': True}
+            surface_info_dict[surface_id] = {'reflector': True}
```

Comment:
This line isn't tested

Reply:
yep this is one of those lines that we can't test without trelis being on the docker image, sorry